pkgs/development/python-modules/web-cache/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "web-cache";
+  version = "1.1.0";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1d8f1s3i0s3h1jqvjq6cp639hhbbpxvyq7cf9dwzrvvvr0s0m8fm";
+  };
+
+  # web_cache doesn't have tests
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Simple Python key-value storage backed up by sqlite3 database";
+    homepage = "https://github.com/desbma/web_cache";
+    license = licenses.lgpl21Only;
```

The licensing is not clear at first glance.

- `setup.py` says [GNU Lesser General Public License v2 or later (LGPLv2+)](https://github.com/desbma/web_cache/blob/master/setup.py#L33)
- Source code entry says [__license__ = "LGPLv2"](https://github.com/desbma/web_cache/blob/master/web_cache/__init__.py#L5)
- License file says [GNU LESSER GENERAL PUBLIC LICENSE Version 2.1](https://github.com/desbma/web_cache/blob/master/LICENSE)
- README says [LGPLv2](https://github.com/desbma/web_cache#license)
- GitHub says `LGPL-2.1 License` (that comes from the license file itself)

For me it looks like that it should be:

```suggestion
    license = licenses.lgpl2Plus;
```

To be sure, ask the upstream project what the actual license is.